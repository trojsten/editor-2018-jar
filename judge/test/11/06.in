STR:
General
OHLVIHZFS
STR:
Plukovnik
DAJBZ
STR:
Podplukovnik
VDUH
STR:
Strateg
CBA
STR:
Spion
OSNPQHGZC
STR:
Programator
OOUME
STR:
Velmajstermaxisupergeneralveduci
KGLAMC
STR:
Nadpodpredzadplukovnikazarovendokoncaajgeneral
OUPZWHQQJB
