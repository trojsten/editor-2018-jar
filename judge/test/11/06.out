{"Velmajstermaxisupergeneralveduci": "OUPZWHQQJB", "Spion": "OOUME", "Nadpodpredzadplukovnikazarovendokoncaajgeneral": "KGLAMC", "Programator": "DAJBZ", "General": "CBA", "Strateg": "OSNPQHGZC", "Podplukovnik": "OHLVIHZFS", "Plukovnik": "VDUH"}